ocs_ci/ocs/platform_nodes.py

```diff
@@ -2167,3 +2167,75 @@ def detach_volume(self, volume, node=None, delete_from_backend=True):
         self.vsphere.remove_disk(
             vm=vm, identifier=volume, key="disk_name", datastore=delete_from_backend
         )
+
+class IBMCloud(NodesBase):
+    """
+    IBM Cloud class
+    """
+
+    def __init__(self):
+        super(IBMCloud, self).__init__()
+        self.ibmcloud = ibmcloud.IBMCloud()
+
+    def test(self):
+        self.ibmcloud.test()
+
+    def stop_nodes(self, nodes, force=True):
+        """
+        Stop PowerNode
+
+        Args:
+            nodes (list): The OCS objects of the nodes
+            force (bool): True for force nodes stop, False otherwise
+
+        """
+        if self.powernodes.iskvm():
+            self.ibmcloud.reboot_node(
+                nodes, timeout=900, wait=True, force=force
+            )
+        else:
+            raise NotImplementedError(
+                "This is not libvirt environment. Stop nodes not implemented"
+            )
+
+    def restart_nodes(self, nodes, timeout=900, wait=True):
+        """
+        Restart ibmcloud vm instances by stop and start
+
+        Args:
+            nodes (list): The OCS objects of the nodes / Azure Vm instance
+            wait (bool): True if need to wait till the restarted node reaches
+                READY state. False otherwise
+            timeout (int): time in seconds to wait for node to reach 'not ready' state,
+                and 'ready' state.
+            force (bool): True for force VM stop, False otherwise
+
+        """
+        self.ibmcloud.restart_nodes(nodes, timeout=900, wait=True)
+
+
+    def restart_nodes_by_stop_and_start(self, nodes, force=True):
+        """
+        Make sure all Nodes are up
+        """
+        self.ibmcloud.restart_nodes_by_stop_and_start(nodes, force=True)
+
+    def attach_volume(self, volume, node):
+        self.ibmcloud.attach_volume(volume, node)
+
+    def detach_volume(self, volume, node=None, delete_from_backend=True):
+        self.ibmcloud.detach_volume(volume, node)
+
+    def get_node_by_attached_volume(self, volume):
+        self.ibmcloud.get_node_by_attached_volume(volume)
+
+    def get_data_volumes(self):
+        self.ibmcloud.get_data_volumes()
+
+    def wait_for_volume_attach(self, volume):
+        self.ibmcloud.wait_for_volume_attach(volume)
+
+    def getVolumeId(self):
+        volumeId = self.ibmcloud.getVolumeId()
+        return volumeId
```

Comment:
```suggestion
        return self.ibmcloud.getVolumeId()
```
if you are not using , you can directly return

Reply:
Done